lib/internal/streams/BufferList.js

```diff
@@ -73,4 +74,8 @@ module.exports = class BufferList {
     }
     return ret;
   }
+
+  [customInspectSymbol]() {
+    return `${this.constructor.name} { length: ${this.length} }`;
```

Comment:
You could `return { __proto__: Object.getPrototypeOf(this), length: this.length }` which would also give coloring etc. for free.

Reply:
Using `__proto__` will actually result in a `call stack size exceeded error. I decided to call inspect directly instead to highlight the color in case it is needed.